go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/aws/karpenter-provider-aws
 
 go 1.22
+toolchain go1.22.2
```

Comment:
FYI to when we merge this. We can just pin the go patch version to ensure that we don't add the toolchain command into the `go.mod`